Dockerfile.boltron

```diff
@@ -0,0 +1,73 @@
+FROM jamesantill/boltron-27
```

Comment:
we should really use original Fedora images here

Reply:
This is the image Irina was supposed to used for the testing. I have double checked that with her. We can update it to a different image when needed.

Reply:
Just saying that this image is not official in any way.